doc/tutorials/installation.rst

```diff
@@ -0,0 +1,142 @@
+======================
+How to install pydicom
+======================
+
+.. note::
+
+   We recommend installing into a
+   `virtual environment <https://docs.python.org/3/tutorial/venv.html>`_,
+   which is an isolated Python environment that allows you to install
+   packages without admin privileges. See the
+   :doc:`virtual environments tutorial<virtualenvs>` on how to create and
+   manage virtual environments.
+
+
+.. _tut_install:
+
+Install the official release
+============================
+
+*pydicom*, being a Python library, requires `Python
+<https://www.python.org/>`_. If you're not sure whether or not your version of
+Python is supported, check :ref:`this table<faq_install_version>`.
+
+Install using pip
+-----------------
+
+*pydicom* is available on `PyPi <https://pypi.python.org/pypi/pydicom/>`_, the
+official third-party Python software repository. The simplest way to install
+from PyPi is using `pip <https://pip.pypa.io/>`_ with the command::
+
+  $ pip install pydicom
+
+You may need to use this instead, depending on your operating system::
+
+  $ python -m pip install pydicom
+
+You can also perform an offline installation by
+`downloading <https://github.com/pydicom/pydicom/releases>`_ and installing
+one of the release ``*.whl`` files. For example, with the v1.3 release::
+
+  $ pip install pydicom-1.3.0-py2.py3-none-any.whl
+
+
+Install using conda
+-------------------
+
+*pydicom* is also available for `conda <https://docs.conda.io/>`_ at
+`conda-forge <https://anaconda.org/conda-forge/pydicom>`_::
+
+  $ conda install -c conda-forge pydicom
+
+
+.. _tut_install_libs:
+
+Install the optional libraries
+------------------------------
+
+If you're going to be manipulating pixel data then we recommend you install
+`NumPy <https://numpy.org/>`_.
+
```

Comment:
I'd say 'recommend' is not strong enough - could say NumPy is required.

Reply:
Done